src/plugins/intel_gpu/tests/test_cases/reorg_yolo_gpu_test.cpp

```diff
@@ -0,0 +1,372 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/activation.hpp>
+#include <intel_gpu/primitives/reorg_yolo.hpp>
+
+#include <cstddef>
+#include <string>
+
+using namespace cldnn;
+using namespace ::tests;
+
+namespace {
+template<typename T>
+struct ReorgYoloParams {
+    ov::PartialShape inputTensor;
+    std::vector<T> input;
+    uint32_t stride;
+    std::vector<T> expected;
+};
+
+template<typename T>
+using ReorgYoloParamsWithLayout = std::tuple<
+    ReorgYoloParams<T>,
+    format::type,       // blocked layout
+    bool                // should_fail
+>;
+
+const std::vector<format::type> dataFormats = {
+    format::bfyx,
+    format::yxfb,
```

Comment:
`format::byxf` is not included intentionally?

Reply:
Done.